Elie.abichar/rumm 3337 improve documentation (#28)
DataDog/dd-sdk-reactnative-examples: rum-expo-react-navigation/README.md

## To build the app

### Locally

> The React Native SDK doesn't run on Expo Go, so to have an honest build, the project needs to be built using `expo-dev-client`. Do that by running:

Reviewer: ```suggestion
The React Native SDK runs separately from Expo Go, which means the project should be built using `expo-dev-client`. Build the project locally by running the following:
```

> We should note that the React native SDK doesn't really run on Expo GO, so to have an honest build, you need to build the project using expo dev client. You can do that by running:

Reviewer 2: ```suggestion
We should note that the React Native SDK doesn't really run on Expo Go, so to have an honest build, you need to build the project using `expo-dev-client`. You can do that by running:
```

> The React Native SDK runs separately from Expo Go, which means the project should be built using `expo-dev-client`. Build the project locally by running the following:

Reviewer: ```suggestion
The React Native SDK runs separately from Expo Go, which means the project must be built using `expo-dev-client`. Build the project locally by running the following:
```

```sh
yarn expo run:android
```

or

```sh
yarn expo run:ios 
```

> ### On eas

Reviewer 2: ```suggestion
### On EAS
```

#### Signing the app

> To sign the app, refer to the [EAS documentation][1].

Reviewer: ```suggestion
Learn how to sign the app in the [EAS documentation][1].
```
